cypress/integration/tenants/add_tenant.spec.js

```diff
@@ -22,13 +22,17 @@ describe('Add Tenant', function () {
   })
 
   it('creates a tenant with a lease', function () {
+    cy.visit('add/property')
+    cy.wait(500)
+    cy.fillInPropertyForm()
+    cy.get('button').contains('SAVE').click()
+    cy.wait(500)
+
     cy.visit('/add/tenant')
     cy.fillInTenantForm()
-    cy.contains('Create New Property').click()
-    cy.fillInPropertyForm()
-    cy.get('form.add-property__form-container').contains('SAVE').click()
-    cy.get('form.add-property__form-container').should('not.exist')
-    cy.contains('The Tenants').should('exist')
+
+    cy.get('input[title="Search Properties"]').first().type('The')
+    cy.get('.styles-module_resultItemLabel__rcaln:contains(The Tenants)').click()
```

Comment:
It looks like this was testing you could create a property when adding a tenant.
This is now testing you can find and select a property when adding a tenant.

I think this is better 👍